### PR TITLE
Cypress/E2E: Updated tests

### PR DESCRIPTION
**Pull request by saturninoabril**
#### Summary
- Updated tests found while reviewing https://github.com/mattermost/mattermost/pull/27098#pullrequestreview-2076366995

#### Ticket Link
none

#### Release Note
```release-note
NONE
```

### CODE REVIEW

File: webapp/platform/types/src/users.ts [hunk 1]
```diff
@@ -39,6 +39,7 @@ export type UserProfile = {
     password: string;
     auth_service: string;
     email: string;
+    email_verified: boolean;
```
Comment (saturninoabril) [inline webapp/platform/types/src/users.ts:42]: This field has been there for so long and probably missed in UserProfile type.

File: e2e-tests/cypress/tests/support/index.d.ts [hunk 2]
```diff
@@ -28,7 +28,7 @@ declare namespace Cypress {
     type Team = import('@mattermost/types/teams').Team;
     type TeamMembership = import('@mattermost/types/teams').TeamMembership;
     type TermsOfService = import('@mattermost/types/terms_of_service').TermsOfService;
-    type UserProfile = import('@mattermost/types/users').UserProfile;
+    type UserProfile = import('@mattermost/types/users').UserProfile & {email_verified: boolean};
```
Comment (saturninoabril) [inline e2e-tests/cypress/tests/support/index.d.ts:31]: Temporary only until the new `@mattermost/types` version is released.

File: e2e-tests/cypress/tests/integration/channels/files_and_attachments/image_link_preview_spec.js [hunk 3]
```diff
@@ -241,12 +241,12 @@ describe('Image Link Preview', () => {
...
-                    expect(imagePreview.height()).to.closeTo(previewSize.height, 1);
-                    expect(imagePreview.width()).to.be.closeTo(previewSize.width, 1);
+                    expect(imagePreview.height()).to.closeTo(previewSize.height, 2);
+                    expect(imagePreview.width()).to.be.closeTo(previewSize.width, 2);
                 } else {
                     // * It should match original dimension for images less than viewport size
-                    expect(imagePreview.height()).to.closeTo(originalSize.height, 1);
-                    expect(imagePreview.width()).to.be.closeTo(originalSize.width, 1);
+                    expect(imagePreview.height()).to.closeTo(originalSize.height, 2);
+                    expect(imagePreview.width()).to.be.closeTo(originalSize.width, 2);
```
Comment (saturninoabril) [inline e2e-tests/cypress/tests/integration/channels/files_and_attachments/image_link_preview_spec.js:249]: Made the limit more forgiving for it does not need to be strict at +/- 1.

File: e2e-tests/cypress/tests/integration/channels/messaging/inline_markdown_image_link_open_link_spec.js [hunk 4]
```diff
@@ -20,7 +20,7 @@ describe('Messaging', () => {
     });
 
     it('MM-T188 - Inline markdown image that is a link, opens the link', () => {
-        const linkUrl = 'https://www.google.com';
+        const linkUrl = 'https://example.com/';
```
Comment (saturninoabril) [inline e2e-tests/cypress/tests/integration/channels/messaging/inline_markdown_image_link_open_link_spec.js:23]: Used the Example page for the previous URL is taking longer to fetch.

**Review by yasserfaraazkhan — APPROVED** [review]
@saturninoabril few spec files failed. I think these are existing failures. 
otherwise, everything looks great.

**Comment by saturninoabril** [comment]
Thanks for checking! I'll take a look again for this PR attempted to fix those tests but still failed.

**Pull request closed without merging**